network/datadog_checks/network/network.py

```diff
@@ -270,7 +270,13 @@ def _is_collect_cx_state_runnable(self, proc_location):
             return False
 
         if proc_location != "/proc":
-            self.warning("Cannot collect connection state: currently with a custom /proc path: %s", proc_location)
+            # If we have `ss`, we're fine with a non-standard `/proc` location
+            try:
+                get_subprocess_output("ss --help", env={"PROC_ROOT": proc_location})
+                return True
+            except OSError as e:
+                self.warning("Cannot collect connection state: `ss` invocation failed: %s. "
+                             "Currently with a custom /proc path: %s", str(e), proc_location)
```

Comment:
If the intent is to verify that `ss` exists ? It seems "expensive" to call ss. Maybe there is a lighter way.

Can this work for this case:

```
>>> import distutils.spawn
>>> distutils.spawn.find_executable("ss")
'/bin/ss'
```

Reply:
Good point.
I’ve implemented your suggestion.